src/frontend/src/utils/iiConnection.ts

```diff
@@ -283,7 +296,8 @@ export class IIConnection {
   }
 
   static async lookupAll(userNumber: UserNumber): Promise<DeviceData[]> {
-    return await baseActor.lookup(userNumber);
+    const actor = await baseActor;
+    return await actor.lookup(userNumber);
```

Comment:
Does this make a difference?

Reply:
See above. This did not fetch root keys correctly (which does not matter for queries, but it does for updates. And for consistency I think it should be fixed everywhere.

Reply:
I'm confused still. Aren't the two following bits equivalent?

``` ts
return await foo.bar();
```

``` ts
const tmp = await foo;
return tmp.bar();
```

Is there something in `const actor =...` that makes it fetch the root key correctly? I might also just be missing something obvious